tools/rna_tools/coprarna/coprarna.xml

```diff
@@ -0,0 +1,123 @@
+            <option value="0" selected="true">Off</option>
+            <option value="1">Organism of interest based consensus</option>
+            <option value="2">Overall consensus based prediction</option>
+    </param>
+    <param argument="--topcount" type="integer" value="200" label="Top predictions" help="Specifies the number of top predictions to return and use for the extended regions plots (def:200)" optional="true"/>
+    <param argument="--cop1" truevalue="--cop1" falsevalue="" checked="False" type="boolean" label="CopraRNA1 prediction" help="Switch for CopraRNA1 prediction (def:off)"/>
+    <param argument="--nooi" truevalue="--nooi" falsevalue="" checked="False" type="boolean" label="No focus on the organism of interest" help="If set then the CopraRNA2 prediction mode is set not to focus on the organism of interest (def:off)"/>
+    <param argument="--enrich" truevalue="--enrich" falsevalue="" checked="False" type="boolean" label="DAVID-WS functional enrichment" help="If set DAVID-WS functional enrichment is calculated with given number of top predictions (def:off)"/>
+    <param argument="--enrich" type="integer" value="0" label="DAVID-WS functional enrichment" help="If set DAVID-WS functional enrichment is calculated with given number of top predictions (def:off)" optional="true"/>
+    <param argument="--root" type="float" value="1" label="Root function" help="Specifies root function to apply to the weights (def:1)"/>
+  </inputs>
+  <outputs>
+    <data format="csv" name="result" from_work_dir="./CopraRNA_result.csv" label="CopraRNA result csv"/>
+    <data format="txt" name="options" from_work_dir="./CopraRNA_option_file.csv" label="CopraRNA option log"/>
+    <data format="pdf" name="conservationheatmap" from_work_dir="./sRNA_conservation_heatmap.pdf" label="sRNA conservation heatmap"/>
+    <data format="pdf" name="functionheatmap" from_work_dir="./Enrichment/enriched_heatmap_big.pdf" label="Functional enrichment heatmap"/>
+    <data format="tsv" name="clusters" from_work_dir="./cluster.tab" label="clusters"/>
+  </outputs>
+  <tests>
+    <test>
+      <param name="srnaseq" value="copra.fa"/>
+      <param name="--ntup" value="75"/>
+      <param name="--ntdown" value="75"/>
+      <param name="--region" value="5utr"/>
+      <param name="--rcsize" value="0.5"/>
+      <param name="--cores" value="1"/>
+      <param name="--winsize" value="150"/>
+      <param name="--maxbpdist" value="100"/>
+      <param name="--enrich" value="200"/>
+      <param name="--cons" value="0"/>
+      <output name="result" file="CopraRNA_result.csv"/>
+      <output name="options" file="CopraRNA_option_file.csv"/>
+      <output name="conservationheatmap" file="sRNA_conservation_heatmap.pdf"/>
+      <output name="clusters" file="cluster.tab"/>
+      <output name="functionheatmap" file="enriched_heatmap_big.pdf"/>
```

Comment:
you can not "diff" a pdf file, so you might want to use another comparison method, like sim_size